app/models/work-item.ts

```diff
@@ -435,14 +435,19 @@ export async function getWorkItemsByUpdateAgeAndStatus(
   workItemStatuses: WorkItemStatus[],
   jobStatuses: JobStatus[],
   fields = tableFields,
+  startingId = 0,
+  batchSize = 1000,
 ): Promise<WorkItem[]> {
   const pastDate = subMinutes(new Date(), lastUpdateOlderThanMinutes);
   const workItems = (await tx(`${WorkItem.table} as w`)
     .innerJoin(Job.table, 'w.jobID', '=', `${Job.table}.jobID`)
     .select(...fields)
     .whereIn(`${Job.table}.status`, jobStatuses)
-    .where('w.updatedAt', '<', pastDate)
-    .whereIn('w.status', workItemStatuses))
+    .where('w.id', '>', startingId)
```

Comment:
This one makes sense to me - you want to track the last ID you received in the prior batch and you don't want to process it again, so use '>'.

Reply:
Oh, I understand the logic - it's the choice of the name 'startingId' I find confusing. I think something like 'lastProcessedId' makes more sense, since `startingId` implies it would be included. Not a big deal, though. Like I said, a nit.

Reply:
I agree the startingId is not included in the `where` clause sounds confusing, but from the function scope and processing flow, the `startingId` is clearer than the `lastProcessedId`. So I will update the comment on startingId to make it clear, but keep the name.